eng/liveBuilds.targets

```diff
@@ -78,10 +78,11 @@
     <CoreCLRCrossTargetComponentDirName Condition="'$(TargetArchitecture)' == 'arm' and '$(BuildArchitecture)' != 'arm' and '$(TargetsLinux)' == 'true'">x64</CoreCLRCrossTargetComponentDirName>
   </PropertyGroup>
 
-  <Target Name="ResolveCoreCLRFilesFromLocalBuild">
-    <Error Condition="!Exists('$(CoreCLRArtifactsPath)')" Text="The CoreCLR artifacts path does not exist '$(CoreCLRArtifactsPath)'. The CoreCLR subset category must be built before building this project." />
+  <Target Name="ResolveVMFilesFromLocalBuild">
```

Comment:
I threw this suggestion out there without much context, so I want to highlight it for other reviewers... "runtime" as CoreCLR/Mono vs. "runtime" as the marketing name of the shared framework has been ambiguous for a while, and at some point early in Mono discussions "VM" came up as a possible less ambiguous name for the CoreCLR/Mono grouping. (The implications of "VM" might actually make it a straight-up incorrect suggestion here, I seem to remember some issue like that with the name, e.g. that aot tooling is included so it doesn't make sense.)

The code has already built up concepts like `RuntimeFlavor` which use the first definition, so maybe it's too late and doesn't matter enough to really change--context clues may be enough.

Reply:
For me, either are fine.  I honestly thought runtime fit better, but I'll go with whatever the group decides.

Reply:
Small preference for `ResolveRuntimeFilesFromLocalBuild` and `RuntimeFiles` because I feel like it has consistency with other property names / item groups we use today. Don't feel particularly strongly about it though.

Reply:
I prefer RuntimeFiles.  Who gets to decide?

Reply:
I put the dotnet/runtime-infrastructure team as a reviewer to discuss/reject the name. I believe the general model is that it's up to your judgment unless repo or area owners care otherwise. So far it sounds like consistency in property naming + not overhauling properties during this PR outweighs the ambiguity, and I don't disagree with that.